tests/test_series.py

```diff
@@ -878,40 +908,58 @@ def test_types_between() -> None:
 
 def test_types_agg() -> None:
     s = pd.Series([1, 2, 3], index=["col1", "col2", "col3"])
-    check(assert_type(s.agg("min"), Any), np.int64)
-    check(assert_type(s.agg(["min", "max"]), pd.Series), pd.Series)
-    check(assert_type(s.agg({"a": "min"}), pd.Series), pd.Series)
-    check(assert_type(s.agg("mean", axis=0), Any), np.float64)
+    check(assert_type(s.agg("min"), int), np.int64)
+    check(assert_type(s.agg(["min", "max"]), "pd.Series[int]"), pd.Series, np.intp)
+    check(assert_type(s.agg({"a": "min"}), "pd.Series[int]"), pd.Series, np.intp)
+    check(
+        assert_type(  # type: ignore[assert-type]
+            s.agg("mean", axis=0), float  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
that's not a good change

Reply:
I think we can either have `s.agg("min")` or `s.agg("mean")` to be correct or go back to `Any`. Adding literal overloads for all possible function names just because of a small difference between int and float sounds like a pain.

Reply:
I think we should go back to `Any` if a string function name is passed.